lacework-agent-windows/values.schema.json

```diff
@@ -391,119 +391,130 @@
                 "priorityClassName",
                 "daemonset"
             ],
-            "properties": {
-                "enable": {
-                    "type": "boolean",
-                    "description": "Enable lacework windows agent"
+          "properties": {
```

Comment:
@hengw0498 why is this indentation change showing up? I probably missed noticing this change earlier.

Reply:
For values.schema.json:
I only added tolerations don't be scared by the lines of changes. It is not a big change.